sdk/python/packages/flet/src/flet/plotly_chart.py

```diff
@@ -1 +1 @@
-from flet.core.plotly_chart import PlotlyChart
+
```

Comment:
This is a proxy to avoid the entire `flet` import failing if Plotly is not installed. Should be here.